service/templates/cloudformation/host/main.yaml

```diff
@@ -0,0 +1,24 @@
+{{define "main"}}AWSTemplateFormatVersion: 2010-09-09
+Description: Main CloudFormation stack.
```

Comment:
I think the description should say this is a host stack. Also it will need the route table entries for the guest cluster scraping.

Reply:
Indeed, description fixed (also the name of the stack object as seen from the console, if both accounts are the same we need to distinguish the stacks). Totally missed the route table entries you mention, will work on them next.

Reply:
Great, thanks! We'll also need the `service` and `endpoint` k8s resources for the guest cluster scraping. It has quite a lot of moving parts! We can do these as later PRs.

Reply:
@rossf7 Host route tables are in place, could you please take a look? This is the template https://github.com/giantswarm/aws-operator/pull/613/files#diff-ed833ccba861d6b81fbcab204ebf07fd and this the adapter https://github.com/giantswarm/aws-operator/pull/613/files#diff-b2ba10ced5c9cb6d1388bf3062a520bf